CMakeLists.txt

```diff
@@ -598,9 +598,9 @@ if(WIN32)
     endif()
 
     if (NOT QUIC_STATIC_LINK_CRT AND NOT QUIC_STATIC_LINK_PARTIAL_CRT)
-        # We are using dynamic linking. Ensure that only the release version of CRT is used.
-        message(STATUS "Configuring for release version of dynamically linked CRT")
-        set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreadedDLL")
+        # We are using dynamic linking. Ensure that only the dynamic CRT is used.
```

Comment:
@csujedihy can you please take a look here? You're the one that added this code to link only release version.

Reply:
Note that you can always just only build the release DLL for redistribution :)

Reply:
cc @csujedihy can you take a quick look at this?